src/main/java/frc/robot/Constants.java

```diff
@@ -48,10 +48,10 @@ public static final class DriveConstants {
         new Translation2d(-kWheelBase / 2, -kTrackWidth / 2));
 
     // Angular offsets of the modules relative to the chassis in radians
-    public static final double kFrontLeftChassisAngularOffset = Math.PI;
-    public static final double kFrontRightChassisAngularOffset = Math.PI;
-    public static final double kRearLeftChassisAngularOffset = 0;
-    public static final double kRearRightChassisAngularOffset = Math.PI;
+    public static final double kFrontLeftChassisAngularOffset = Math.PI + (Math.PI/2);
```

Comment:
Review these constants

Reply:
Add this comment:

These are the offsets to get the absolute encoders aligned with the drive train. They are offset from the default offsets from the rev swerve alignment tool. DO NOT CHANGE THESE VALUES UNLESS YOU KNOW WHAT YOU ARE DOING!